### PR TITLE
Update docs theme 1.2.1

### PR DESCRIPTION
**Pull request by dgarcia360**
Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/310

ScyllaDB Sphinx Theme 1.2.1 is now released 🥳 

We’ve added automatic checks for broken links and introduced numerous UI updates.

You can read more about all notable changes [here](https://sphinx-theme.scylladb.com/stable/upgrade/CHANGELOG.html#march-2022).

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

4. Open http://127.0.0.1:5500/ with your favorite browser. The doc should render without errors, and the version should be Sphinx Theme version (see the footer) must be ``1.2.x``:

image

### CODE REVIEW

**Comment by avelanarius** [comment]
@dgarcia360 As you have surely seen, after this PR GitHub Actions job building docs (as well as other jobs) fails with:

```
 Error: error: unable to create symlink docs/source/index.md: File name too long
```

**Comment by dgarcia360** [comment]
@avelanarius Thanks, fixed! It's now ready for review.

**Comment by annastuchlik** [comment]
@avelanarius Is it possible for you to verify this PR on Linux and merge it if it's ok?

File: .github/workflows/docs-pages.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: "Docs / Publish"
```
Comment (avelanarius) [inline .github/workflows/docs-pages.yaml:1]: Why the naming change from `docs-pages@v2` to `docs-pages`?

File: README-dev.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+The docs build instructions have been tested with Sphinx 4 and Fedora 32.
+
+## Prerequisites
+
+To build and preview the docs locally, you will need to install the following software:
+
+- Git
+- Python 3.7
+- pip
+- Java JDK 6 or above
```
Comment (avelanarius) [inline README-dev.md]: I don't think you can successfully build with JDK 6. The code itself is JDK 6 compatible, but due to Maven Central Repository requiring specific TLS standard (unsupported in JDK 6) fetching dependencies will fail.

File: .github/workflows/docs-pages.yaml (hunk 1 above)
Comment (dgarcia360) [reply .github/workflows/docs-pages.yaml:1]: We received feedback from other projects they prefer the workflows files not to be versioned. Therefore, we applied this recommendation in the main repo https://github.com/scylladb/sphinx-scylladb-theme

https://github.com/scylladb/sphinx-scylladb-theme/pull/348

File: README-dev.md (hunk 2 above)
Comment (dgarcia360) [reply README-dev.md]: This comes from the README:

> The java driver supports Java JDK versions 6 and above.

I can confirm it is building with JDK 8: https://github.com/scylladb/java-driver/blob/scylla-3.x/.github/workflows/docs-pages%40v2.yaml#L28

```suggestion
- Java JDK 8 or above
```

**Comment by avelanarius** [comment]
Merged via 8feeb205fcf163d9a45dbe884fc353ef5a1a4535 (squashed commits into one).

**Comment by annastuchlik** [comment]
@dgarcia360 @avelanarius Thank you!

**Pull request closed without merging**